Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -132,7 +148,11 @@
     private Directory index;
     private Path relativePath = null;
     private IndexSearcher searcher = null;
-
+    private Analyzer customAnalyzer() throws IOException{
+        return CustomAnalyzer.builder().withTokenizer("standard").addTokenFilter("asciifolding").
+                addTokenFilter("lowercase").build();
+    }
```

Comment:
is there a reason that this is a method rather than just a class variable?

Reply:
when declaring it as the class variable its throwing the error "While constructing the default constructor can not handle exception : type Exception thrown by implicit super constructor". To handle the exception I used method instead of a class variable.

Reply:
Initialize a class variable analyzer in an init method and you should be fine.